app/assets/javascripts/discourse/app/templates/modal/change-owner.hbs

```diff
@@ -12,7 +12,8 @@
     {{user-selector single="true"
                     usernames=new_user
                     placeholderKey="topic.change_owner.placeholder"
-                    autocomplete="discourse"}}
+                    autocomplete="discourse"
+                    autofocus="autofocus"}}
```

Comment:
```suggestion
                    autofocus=true}}
```

(though it's nothing more than a code-golf suggestion; both variants are rendered as `<input autofocus …>`)